server/src/alda/lisp/attributes.clj

```diff
@@ -14,15 +14,28 @@
   {:pre [(<= 0 x)]}
   (constantly (/ x 100.0)))
 
+;; Validation that the input is an integer value
+(defn- pos-integer[x]
```

Comment:
There should be a space between the function name and the vector of arguments: `pos-integer [x]`